docs/extend/install-configure-zos-extensions.md

```diff
@@ -44,6 +44,40 @@ $ zowe-install-component.sh \
     -l /var/zowe/instance/logs
 ```
 
+## Upgrade with `zowe-upgrade-component.sh` (Technical Preview)
+
+<Badge text="Technical Preview"/>
+
+_Note: This section is for technical preview and we are happy to hear any feedback. Content in this section may be changed or improved in the future._
+
+_Note: this feature is added with Zowe v1.21.0 release._
+
+From Zowe v1.21.0, Zowe ships a `bin/zowe-upgrade-component.sh` tool to help you upgrade any Zowe server component (extension) to the latest version available into the Zowe Artifactory. 
```

Comment:
Should be v1.22.0.

Reply:
Think the script only works for core components for now, so thinking to make it clear that it doesn't work for extensions.

Reply:
Thanks Jack, I will fix it!